sr_ur_arm_calibration_loader/src/sr_ur_arm_calibration_loader/sr_ur_arm_calibration_loader.py

```diff
@@ -81,12 +81,23 @@ def _setup_folders(self):
     def _get_serial_from_arm(self, arm_ip):
         client = paramiko.SSHClient()
         client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
-        client.connect(arm_ip, username=self._CONST_UR_ARM_SSH_USERNAME, password=self._CONST_UR_ARM_SSH_PASSWORD)
-        stdin, stdout, stderr = client.exec_command('cat /root/ur-serial')
-        arm_serial_number = stdout.readline()
-        client.close()
+        arm_serial_number = ''
+        ssh_exception_message = ''
+        try:
+            client.connect(arm_ip, username=self._CONST_UR_ARM_SSH_USERNAME,
+                           password=self._CONST_UR_ARM_SSH_PASSWORD, timeout=5.0)
+            stdin, stdout, stderr = client.exec_command('cat /root/ur-serial')
+            arm_serial_number = stdout.readline()
+            client.close()
+        except (paramiko.BadHostKeyException, paramiko.AuthenticationException, paramiko.SSHException):
+            ssh_exception_message = "Failed to SSH into arm"
```

Comment:
I wonder if it would be useful to include actual exception message into `ssh_exception_message`?

Reply:
Sure, for me makes sense to have more info regarding the exception